### PR TITLE
[CI] Fix asan+ubsan build

### PR DESCRIPTION
**Pull request by Flakebi**
- Change `LD_PRELOAD` to `LD_LIBRARY_PATH`, which fixes building gpurt with asan+ubsan enabled.
- Enable sanitizer builds for the image, enabling it for PRs will come later, when the images are built and working.
- Disable coverage builds, the comments are gone and I don’t think anybode looks at their results (except for passing).
- Move `amdvlkadmin/` into the `image-template` to reduce duplication of the image path. This gets rid of the `DOCKER_USR` secret, which wasn’t that secret anyway.

### CODE REVIEW

File: .github/workflows/check-amdllpc-docker.yml
Comment (qiaojbao) [inline .github/workflows/check-amdllpc-docker.yml:19]: I think you should add all the build types like asan_ubsan here too.

Comment (Flakebi) [reply .github/workflows/check-amdllpc-docker.yml:19]: You mean below, in the `feature-set`?
Yes, I plan to do that, but I need to wait until this PR is merged and the base image is built once and uploaded, otherwise the asan+ubsan `check-amdllpc-docker` jobs will fail because they can’t find the base image.

**Comment by amdvlk-admin** [comment]
### Test summary for commit d7615308b8d053a3d9d82b03f093d3fcb0714fc6
<details><summary>CTS tests (Failed: 1/207274)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi2x</summary><ul>

<li>Passed: 38904/70473 (55.2%)</li>
<li><details><summary>Failed: 1/70473 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.binding_model.descriptorset_random.sets8.constant.ubolimitlow.sbolimitlow.sampledimghigh.outimgonly.iublimitlow.uab.frag.ialimitlow.0
Stack trace: Flake
```
</p></details></li>

<li>Not Supported: 31568/70473 (44.8%)</li><li>Warnings: 0/70473 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 35750/70481 (50.7%)</li>
<li>Failed: 0/70481 (0.0%)</li>
<li>Not Supported: 34731/70481 (49.3%)</li><li>Warnings: 0/70481 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 9.0, Gfx9</summary><ul>

<li>Passed: 33476/66320 (50.5%)</li>
<li>Failed: 0/66320 (0.0%)</li>
<li>Not Supported: 32844/66320 (49.5%)</li><li>Warnings: 0/66320 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by Flakebi** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit d7615308b8d053a3d9d82b03f093d3fcb0714fc6
<details><summary>CTS tests (Failed: 0/207274)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi2x</summary><ul>

<li>Passed: 38905/70473 (55.2%)</li>
<li>Failed: 0/70473 (0.0%)</li>
<li>Not Supported: 31568/70473 (44.8%)</li><li>Warnings: 0/70473 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 35750/70481 (50.7%)</li>
<li>Failed: 0/70481 (0.0%)</li>
<li>Not Supported: 34731/70481 (49.3%)</li><li>Warnings: 0/70481 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 9.0, Gfx9</summary><ul>

<li>Passed: 33476/66320 (50.5%)</li>
<li>Failed: 0/66320 (0.0%)</li>
<li>Not Supported: 32844/66320 (49.5%)</li><li>Warnings: 0/66320 (0.0%)</li>
</ul></details>


</ul></details>

**Review by qiaojbao — APPROVED** [review]

**Review by amdrexu — APPROVED** [review]

**Comment by qiaojbao** [comment]
Seems the docker build still failed.

**Comment by Flakebi** [comment]
The ubsan failure is fixed by #2160.

**Pull request merged**
